pkg/gadget-context/gadget-context.go

```diff
@@ -119,6 +122,10 @@ func (c *GadgetContext) GadgetParams() *params.Params {
 	return c.gadgetParams
 }
 
+func (c *GadgetContext) GadgetParamsWOFlag() []string {
+	return c.gadgetParamsWOFlag
+}
```

Comment:
I think it can be just `GadgetArgs()`

Reply:
Hah, I was just going to write the same :) Maybe even just `Arguments()` or `Args()` as `Gadget` is already in `GadgetContext` and IMHO doesn't need to be more specific like the params.